src/asynchronous/events.d

```diff
@@ -175,8 +175,34 @@ unittest
     assert(exceptionContext.toString.canFind("future"));
 }
 
+/**
+ * An SSL context holds various data longer-lived than single SSL connections,
+ * such as SSL configuration options, certificate(s) and private key(s).
+ */
 interface SslContext
 {
+	/**
+	 * Load a set of "certification authority" (CA) certificates used to validate
+	 * other peers' certificates.
+	 *
+	 * Params:
+	 *  CAFile = the path to a file of concatenated CA certificates in PEM format.
+	 */
+	void loadVerifyLocations(in string CAFile);
```

Comment:
The problem is if I remove ```in``` I can't use ```in``` in the implementation and I think that these functions should never change the passed values. So it should be ok to force these values to be constant in the interface. Should I remove ```in``` anyway?